app/src/app/modules/main-page/main-page-routing.module.ts

```diff
@@ -1,8 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ProfileComponent } from './components/profile/profile.component';
 import { MainPageComponent } from './main-page.component';
 
-const routes: Routes = [{ path: '', component: MainPageComponent }];
+const routes: Routes = [{
+  path: '', component: MainPageComponent
+},
+{
+  path: 'profile', component: ProfileComponent
+}];
```

Comment:
Lepiej to dać jako:
```
const routes: Routes = [
  {
    path: '',
    component: MainPageComponent,
    children: [
      {
        path: 'profile',
        component: ProfileComponent
      }
    ]
  }
];
```
Bo mainPageComponent to root komponent ktory ma w sobie dzieci czyli w tym wypadku profile. I lepiej w taki sposób implementować routy bo można dodać w głównym module <router-outlet></router-outlet> który na bieżąco odpowiednie rzeczy podmienia (Navbar jest statyczny więc go nie trzeba ani podmieniać)